templates/base.html

```diff
@@ -0,0 +1,129 @@
+{% load static %}
```

Comment:
Put new changes in a new PR. Only make changes that were requested for or relevant to the PR. Put all these changes in a new PR